src/mapper/pkg/resolvers/schema.resolvers.go

```diff
@@ -21,40 +21,63 @@ import (
 	"golang.org/x/exp/slices"
 )
 
-func (r *mutationResolver) ResetCapture(ctx context.Context) (bool, error) {
+func (r *mutationResolver) ResetCapture(_ context.Context) (bool, error) {
 	logrus.Info("Resetting stored intents")
 	r.intentsHolder.Reset()
 	return true, nil
 }
 
-func (r *mutationResolver) ReportCaptureResults(ctx context.Context, results model.CaptureResults) (bool, error) {
-	for _, captureItem := range results.Results {
-		srcPod, err := r.kubeFinder.ResolveIpToPod(ctx, captureItem.SrcIP)
-		if err != nil {
-			if errors.Is(err, kubefinder.ErrFoundMoreThanOnePod) {
-				logrus.WithError(err).Debugf("Ip %s belongs to more than one pod, ignoring", captureItem.SrcIP)
-			} else {
-				logrus.WithError(err).Debugf("Could not resolve %s to pod", captureItem.SrcIP)
-			}
-			continue
+func (r *mutationResolver) discoverSrcIdentity(ctx context.Context, src model.RecordedDestinationsForSrc) *model.OtterizeServiceIdentity {
```

Comment:
Annoying Grapql SDK thing - this function will get deleted if you run `go generate` twice. At the first time, you'll get a warning comment. It just needs to move to `helpers.go` file